### PR TITLE
bees: 0.6.2 -> 0.6.3

### PR DESCRIPTION
**Pull request by priegger**
###### Motivation for this change

Update bees to the latest version: https://github.com/Zygo/bees/releases

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/filesystems/bees/default.nix [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 let
 
-  version = "0.6.2";
-  sha256 = "05niv9rivd3j3cwcx3n3vjr85wr0l5i76giq9n54d2vdwmn8qjib";
+  version = "0.6.3";
+  hash = "sha256-brEjr7lhmKDCIDeLq+XP+ZTxv1RvwoUlszMSEYygxv8=";
```
Comment (mweinelt) [inline pkgs/tools/filesystems/bees/default.nix:6]: Any particular reason those two are outside stdenv.mkDerivation?

**Comment by mweinelt** [comment]
Result of `nixpkgs-review pr 100257` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>bees</li>
  </ul>
</details>

File: pkgs/tools/filesystems/bees/default.nix (hunk 1 above)
Comment (priegger) [reply pkgs/tools/filesystems/bees/default.nix:6]: I guess it's to have the "moving parts" together and at the file (it was like this before). What's your recommendation/preference?

File: pkgs/tools/filesystems/bees/default.nix [hunk 2]
```diff
@@ -13,7 +13,7 @@ let
       owner = "Zygo";
       repo = "bees";
       rev = "v${version}";
-      inherit sha256;
+      inherit hash;
```
Comment (mweinelt) [inline pkgs/tools/filesystems/bees/default.nix:16]:
```suggestion
      sha256 = "sha256-brEjr7lhmKDCIDeLq+XP+ZTxv1RvwoUlszMSEYygxv8=";
```

File: pkgs/tools/filesystems/bees/default.nix [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 let
 
-  version = "0.6.2";
-  sha256 = "05niv9rivd3j3cwcx3n3vjr85wr0l5i76giq9n54d2vdwmn8qjib";
+  version = "0.6.3";
+  hash = "sha256-brEjr7lhmKDCIDeLq+XP+ZTxv1RvwoUlszMSEYygxv8=";
 
   bees = stdenv.mkDerivation {
     pname = "bees";
```
Comment (mweinelt) [inline pkgs/tools/filesystems/bees/default.nix:9]:
```suggestion
    pname = "bees";
    version = "0.6.3";
```

**Review by mweinelt — COMMENTED** [review]
This is how I usually see it done.

File: pkgs/tools/filesystems/bees/default.nix (hunk 2 above)
Comment (priegger) [reply pkgs/tools/filesystems/bees/default.nix]: Done, thanks.

File: pkgs/tools/filesystems/bees/default.nix (hunk 1 above)
Comment (priegger) [reply pkgs/tools/filesystems/bees/default.nix]: I added a second commit fixing this. I can squash the commits, if that's preferred.

File: pkgs/tools/filesystems/bees/default.nix [hunk 4]
```diff
@@ -2,18 +2,15 @@
...
-    inherit version;
+    version = "0.6.3";
 
     src = fetchFromGitHub {
       owner = "Zygo";
       repo = "bees";
       rev = "v${version}";
-      inherit sha256;
+      hash = "sha256-brEjr7lhmKDCIDeLq+XP+ZTxv1RvwoUlszMSEYygxv8=";
```
Comment (mweinelt) [inline pkgs/tools/filesystems/bees/default.nix:13]:
```suggestion
      sha256 = "sha256-brEjr7lhmKDCIDeLq+XP+ZTxv1RvwoUlszMSEYygxv8=";
```

File: pkgs/tools/filesystems/bees/default.nix [hunk 5]
```diff
@@ -58,7 +55,7 @@ let
 
 in
 
-runCommand "bees-service-${version}" {
```
Comment (mweinelt) [inline pkgs/tools/filesystems/bees/default.nix:61]: Hm, I wonder what the point of this was. This would explain why it was outside of stdenv.mkDerivation.

Comment (priegger) [reply pkgs/tools/filesystems/bees/default.nix:61]: Yes. I dropped the version here in a separate commit. I'd rather drop it from the PR and keep version and hash outside of stdenv.mkDerivation, but both is perfectly fine for me.

**Comment by mweinelt** [comment]
I'm rolling out that change locally to test it. Meanwhile the actual maintainer @charles-dyfis-net should do a review.

**Review by charles-dyfis-net — APPROVED** [review]

**Comment by charles-dyfis-net** [comment]
> I'm rolling out that change locally to test it. Meanwhile the actual maintainer @charles-dyfis-net should do a review.

LGTM! (Thank you for taking care of this -- my apologies for not catching the release and doing the update myself; I've been running a build off master, and thus wasn't watching the stable branch closely).

**Pull request merged**